stories/pill/pill.stories.mdx

```diff
@@ -0,0 +1,141 @@
+import { Canvas, Meta, Preview, Story } from "@storybook/addon-docs";
+import { PlusCircleFillIcon } from "@lifesg/react-icons/plus-circle-fill";
+import { PlaceholderIcon } from "@lifesg/react-icons/placeholder";
+
+import { Pill } from "src/pill";
+import { Secondary, Heading3, Title } from "../storybook-common";
+import { Grid, Label, Row, Wrapper } from "./doc-elements";
+import { PropsTable } from "./props-table";
+
+<Meta title="Modules/Pill" component={Pill} />
+
+<Title>Pill</Title>
+
+<Secondary>Overview</Secondary>
+
+A status pill component.
+
+Example of usage: As a status indicator on current flow state such as `Pending Approval`, `Approved`, `Rejected` and etc.
+
+```tsx
+import { Pill } from "@lifesg/react-design-system/pill";
+```
+
+The `Pill` component comes in 2 type, a `solid` and `outline`.
+
+<Canvas>
+    <Story name="Pill">
+        <Wrapper>
+            <Pill type="outline" colorType="blue">
+                Pending Approval
+            </Pill>
+            <br />
+            <Pill type="outline" colorType="green">
+                Approved
+            </Pill>
+            <br />
+            <Pill type="solid" colorType="red">
+                Rejected
+            </Pill>
+        </Wrapper>
+    </Story>
+</Canvas>
+
+Example of `Pill` usage with icon.
```

Comment:
Add a `Heading3` titled "Pill with icon" and this description is no longer needed